<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="HW1650" name="MyRunner.CopyOfTestRunner" tests="1" failures="0" timestamp="04 Apr 2019 15:52:52 GMT" time="23.927" errors="1">
  <testcase name="feature" time="23.927" classname="MyRunner.CopyOfTestRunner">
    <error type="cucumber.runtime.CucumberException" message="java.lang.NullPointerException">
      <![CDATA[cucumber.runtime.CucumberException: java.lang.NullPointerException
at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)
at MyRunner.CopyOfTestRunner.feature(CopyOfTestRunner.java:43)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NullPointerException
at reusableActions.e2e_Scenario.verifyHomePage(e2e_Scenario.java:96)
at stepDefinitions.CreateNewDriver.getSwiftHomePage(CreateNewDriver.java:29)
at âœ½.And Verify home pages(src/main/java/Features/CreateNewDriver.feature:5)
]]>
    </error>
  </testcase> <!-- feature -->
</testsuite> <!-- MyRunner.CopyOfTestRunner -->
